include/mesh/dim2/materials/materials.hpp

```diff
@@ -50,14 +50,15 @@ template <> struct materials<specfem::dimension::type::dim2> {
             specfem::element::property_tag property>
   struct material {
     int n_materials; ///< Number of elements
-    std::vector<specfem::medium::material<type, property> >
+    std::vector<specfem::medium::material<dimension, type, property> >
```

Comment:
```suggestion
    std::vector<specfem::medium::material<dimension_tag, type, property> >
```
Needs updating above and below